homeassistant/components/rss_feed_template.py

```diff
@@ -0,0 +1,93 @@
+"""
+Exports sensor values via RSS feed.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/rss_feed_template/
+"""
+
+import asyncio
+from html import escape
+from aiohttp import web
+
+import voluptuous as vol
+
+from homeassistant.components.http import HomeAssistantView
+import homeassistant.helpers.config_validation as cv
+
+DOMAIN = "rss_feed_template"
+DEPENDENCIES = ['http']
+
+CONTENT_TYPE_XML = "text/xml"
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: cv.ordered_dict(
+        vol.Schema({
+            vol.Optional('title'): cv.template,
+            vol.Required('items'): vol.All(
+                cv.ensure_list,
+                [{
+                    vol.Optional('title'): cv.template,
+                    vol.Optional('description'): cv.template,
+                }]
+                )
+            })
+        )
+    }, extra=vol.ALLOW_EXTRA)
+
+
+def setup(hass, config):
+    """Setup the RSS feeds."""
+    for (feeduri, feedconfig) in config[DOMAIN].items():
+        rss_view = RssView(hass, feedconfig)
+        hass.http.app.router.add_route('get',
+                                       '/api/rss/%s' %
```

Comment:
Sure. Should I have it at /rss_template instead of /api/rss_template? Seems that the UI overrides everything outside /api and I found no way to put it there.